cmd/jaeger/internal/integration/grpc_test.go

```diff
@@ -0,0 +1,92 @@
+// Copyright (c) 2024 The Jaeger Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+package integration
+
+import (
+	"log"
+	"os"
+	"testing"
+	"time"
+
+	"github.com/open-telemetry/opentelemetry-collector-contrib/testbed/testbed"
+	"github.com/stretchr/testify/require"
+
+	"github.com/jaegertracing/jaeger/cmd/jaeger/internal"
+	"github.com/jaegertracing/jaeger/cmd/jaeger/internal/extension/jaegerstorage"
+	"github.com/jaegertracing/jaeger/cmd/jaeger/internal/integration/datareceivers"
+	grpcCfg "github.com/jaegertracing/jaeger/plugin/storage/grpc/config"
+)
+
+func TestGRPCStorage(t *testing.T) {
```

Comment:
This looks like way too much boiler plate that I assume will be common across all storage tests. Can we refactor to make it reusable?